telegram/_chat.py

```diff
@@ -134,6 +134,10 @@ class Chat(TelegramObject):
             :meth:`telegram.Bot.get_chat`.
 
             .. versionadded:: 20.0
+        has_restricted_voice_and_video_messages (:obj:`bool`, optional): :obj:`True`, if the
+            privacy settings of the other party restrict sending voice and video note messages
+            in the private chat. Returned only in :meth:`telegram.Bot.get_chat`.
```

Comment:
```suggestion
            in the private chat. Returned only in :meth:`telegram.Bot.get_chat`.
            
            .. versionadded:: 20.0
```

Reply:
That is already there, its just hidden in the git diff.